src/controls/shared/WidgetWithLabelUI.tsx

```diff
@@ -46,7 +46,7 @@ export const WidgetWithLabelUI = observer(function WidgetWithLabelUI_(p: {
             tw={[
                 'bg-base-100',
                 //
-                WidgetBlockUI && 'WIDGET-WITH-BLOCK',
+                isCollapsible && 'WIDGET-WITH-BLOCK',
```

Comment:
we probably need to either change all those weird names like 'WIDGET-WITH-BLOCK', because right now, those conditions do not really make sense on the semantic level; but it makes sense to have done it that way